client/js/Deck.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-unused-vars */
+/*
+ suits, values(ranks) declared
+ */
+const suits = ['spades', 'clubs', 'hearts', 'diams'] // Array of Card Suits
+const values = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'] // Array of possible card-ranks ie. values.
```

Comment:
Do we need the 'suits' and 'values' in `bluff.js` now? Maybe move it to constructor of class Deck in `Deck.js` ?